flowcraft/generator/templates/megahit.nf

```diff
@@ -32,38 +32,39 @@ process megahit_{{ pid }} {
     template "megahit.py"
 
 }
-IN_fastg = Channel.create()
 
 fastg = params.fastg{{ param_id }} ? "true" : "false"
-if (fastg) {
-    process megahit_fastg_{{ pid }}{
+process megahit_fastg_{{ pid }}{
```

Comment:
Why did you remove the existing conditional that would executed the fastg stuff only when the param was provided?

Reply:
When I tested while I was implementing kraken2, it wasn't working. The process was being executed regardless of the value of the conditional. So I changed it.

Reply:
So, in this way both options work fine?

Reply:
yes. I've tested with both options